roles/sushy_emulator/tasks/collect_details.yml

```diff
@@ -25,7 +25,7 @@
 
 - name: Set sushy url for ocp installation
   ansible.builtin.set_fact:
-    _sushy_url: "http://sushy-emulator.{{ _ingress_domain.stdout }}"
+    ocp_sushy_uri: "http://sushy-emulator.{{ _ingress_domain.stdout }}"
```

Comment:
creating a new internal thing - needed because we can't override parameters via set_fact...